sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample_ManageOperationsAsync.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.DocumentAnalysis.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.DocumentAnalysis.Samples
+{
+    public partial class DocumentAnalysisSamples : SamplesBase<DocumentAnalysisTestEnvironment>
+    {
+        [Test]
+        public async Task ManageOperationsAsync()
```

Comment:
nit: Manage could suggest the users could create/delete/update any of the operations.

Reply:
Updated to GetAndListOperations